.github/scripts/trymerge.py

```diff
@@ -341,6 +339,13 @@ def fetch_json(url: str,
         url += '?' + '&'.join(f"{name}={urllib.parse.quote(str(val))}" for name, val in params.items())
     return cast(List[Dict[str, Any]], _fetch_url(url, headers=headers, data=data, reader=json.load))
 
+def fetch_json_dict(url: str,
```

Comment:
There's also duplication in the code here and in fetch_json-->could we refactor that?